modules/ROOT/pages/markdown.adoc

```diff
@@ -0,0 +1,102 @@
+= Markdown plugin
+:navtitle: Markdown
+:description: The Markdown Premium plugin detects pure markdown from a paste event within a TinyMCE editor instance.
+:description_short: Convert pure markdown on paste.
+:keywords: plugin, Markdown.
+:pluginname: Markdown
+:plugincode: markdown
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The {pluginname} Premium plugin detects pure markdown from a paste event within a {productname} editor instance.
+
+== Using the {pluginname} plugin
+
+=== How it works
+
+The **Markdown** Premium Plugin allows the editor to detect if pure markdown is pasted into the {productname} editor instance.
+
+During this paste event, If the editor detects markdown has been pasted it will:
+
+* attempt to convert the markdown to HTML, while
+* add a undo level before the conversion to allow for undo of plain-text.
+
+=== Basic Syntax
+
+[NOTE]
+These syntax elements listed in the table below represent the current supported Markdown syntax for the initial release.
+
+.Markdown syntax (Basic)
+[cols="1,3",options="header"]
+|===
+| Element | Markdown Syntax
+| Heading | `+# H1+`
+| | `+## H2+`
+| | `+### H3+`
+| Bold | `+**bold text**+`
+| Italic | `+*italicized text*+`
+| Blockquote | `+> blockquote+`
+| Ordered List | `+1. First item+`
+| | `+2. Second item+`
+| | `+3. Third item+`
+| Unordered List | `+- First item+`
+| | `+- Second item+`
+| | `+- Third item+`
+| Code | `++`code`++`
+| Horizontal Rule | `+---+`
+| Link | `+[title](https://www.tiny.cloud/tinymce)+`
+| Image | `+![alt text](image.jpg)+`
+|===
```

Comment:
@vpyshnenko @HAFRMO Ive added a table to outline what we support for initial release. FYI

Reply:
@kemister85  After reviewing the docs, our product team feels we can skip the "Basic Syntax" section. Instead, let's add a "Tip" section to include handy shortcuts for copying and pasting in plain text format. Let's chat.

Reply:
This has been removed, and replaced with new `handy shortcuts` as discussed. "please review new updates"